server/api/userRouter.js

```diff
@@ -0,0 +1,44 @@
+const User = require('../db/models/user');
+const express = require('express');
+const router = express.Router();
+
+/**
+ * Responsds with the mongoId of a user based on their google account id.
+ * If the user doesn't exist in mongo user db, new user created in mongo.
+ * 
+ * req.body: {googleId: String}
```

Comment:
Does documentation match the implementation? req.body and req.params are different.

Reply:
yes did not match- fixed!